src/tokenomics.md

```diff
@@ -49,24 +49,30 @@ head:
 
 In this section you will find all the information on SORA
 tokenomics. Starting with the Token Bonding curve, each subsection
-will cover the main [SORA economy](sora-economy.md) tokens. It is
-important to remember that the [XOR](xor.md) token does not follow
+will cover the main [SORA economy](/sora-economy.md) tokens. It is
+important to remember that the [XOR](/xor.md) token does not follow
 conventional tokenomics of limited supply, instead, its supply is
-elastic, tied to the [Token Bonding Curve (TBC)](tbc.md) and managed
-through [SORA governance](sora-governance.md).
+elastic, tied to the [Token Bonding Curve (TBC)](/tbc.md) and managed
+through [SORA governance](/sora-governance.md).
 
-- The [Token Bonding Curve (TBC)](tbc.md) is the smart contract that
+- The [Token Bonding Curve (TBC)](/tbc.md) is the smart contract that
   manages the supply of XOR in a rational way, without the involvment
   of humans.
-- [XOR](xor.md) is the network utility token used for transaction fees (gas).
-- [VAL](val.md) is the token awarded to validators for securing the
+- [XOR](/xor.md) is the network utility token used for transaction fees (gas).
+- [VAL](/val.md) is the token awarded to validators for securing the
   SORA network.
-- [PSWAP](pswap.md) is the LP reward token for liquidity providers on
+- [PSWAP](/pswap.md) is the LP reward token for liquidity providers on
   [Polkaswap](https://polkaswap.io/) and is burnt with every transaction.
-- [XST](xst.md) is the _platform token_ used as the collateral asset for the creation of SORA Synthetics.
-- The [Token Bonding Curve Dollar (TBCD)](tbcd.md) is an algorithmic,
+- [XST](/xst.md) is the _platform token_ used as the collateral asset for the creation of SORA Synthetics.
+- The [Token Bonding Curve Dollar (TBCD)](/tbcd.md) is an algorithmic,
   non-synthetic stablecoin that has its value maintained by the SORA
   token bonding curve.
+- The [PUSSY](/pussy.md) token is a dedicated meme cryptocurrency in the SORA economy aimed
```

Comment:
@WRRicht3r I don't think we need to specifically mention the PUSSY token since it will be changed over time. 
We can mention that there will be a rotating token roster, and include it as the first one.